.circleci/config.yml

```diff
@@ -0,0 +1,52 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: linkerd/rustup-nightly:v1
+
+    working_directory: ~/tacho
+    steps:
+      - run:
+          name: update rust nightly
+          command: rustup update nightly
+
+      - restore_cache:
+          key: rust-{{ checksum "/rust/update-hashes/nightly-x86_64-unknown-linux-gnu" }}.0
+
+      - run:
+          name: install rustfmt and clippy
+          command: |
+            if [ ! -x /cargo/bin/cargo-fmt ]; then cargo install rustfmt ; fi
+            if [ ! -x /cargo/bin/cargo-clippy ]; then cargo install clippy ; fi
+
+      - save_cache:
+          key: rust-{{ checksum "/rust/update-hashes/nightly-x86_64-unknown-linux-gnu" }}.0
+          paths:
+            - /rust
+            - /cargo
```

Comment:
Yeah, it's basically the best we can do since we don't have anything to key on for clippy/rustfmt.  Since we're relying on a nightly rust sha, I think this is basically good enough (i.e. we'll catch clippy/rustfmt changes the next day).